assets/app/styles/_components.less

```diff
@@ -58,6 +58,13 @@
       .component-label-details {
         text-transform: none;
       }
+      a:not(.keep-link-style) {
```

Comment:
It feels a little weird to take away styles with a CSS class. I prefer it when the rules are additive. Is the only place we're using `keep-link-style` the pods link?